10up/wp-scaffold
# Feature/10up toolkit 3.0.0

### Description of the Change




Closes #

This updates `wp-scaffold` to work with 10up-toolkit@3.0.0 and HMR & Fast Refresh.

It basically removes all css entry points from `package.json` and imports them via a corresponding js file.

### Alternate Designs



### Possible Drawbacks



### Verification Process



- Clone this branch.
- Make sure to remove node_modules and start from a clean state.
- Run `npm install` from the root
- Set SCRIPT_DEBUG to true
- `npm run watch`
- Edit any CSS file (admin-style.css, frontend.css, editor-styles-overrides.css, styleguide.css etc...) Changes should hot reload
- Edit the example block css and react code. Changes should hot reload.
- 
### Checklist:






### Changelog Entry



### Credits


Props @
